wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Content/Text/ConversationLinkAttachmentCell.swift

```diff
@@ -152,6 +152,8 @@ final class ConversationLinkAttachmentCellDescription: ConversationMessageCellDe
     weak var delegate: ConversationMessageCellDelegate?
     weak var actionController: ConversationMessageActionController?
 
+    var canBeCombinedWithOtherCells: Bool { true }
```

Comment:
I wasn't able to manually test this, how can a `ConversationLinkAttachmentCell` be shown?